internal/rollup/uglify-wrapped.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license
+ * Copyright 2017 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// A wrapper around uglify-js that handle minifies directories as inputs
+// and outputs
+
+const fs = require('fs');
+const path = require('path');
+const child_process = require('child_process');
+
+const DEBUG = false;
+
+// capture the inputs and output options
+const argv = require('minimist')(process.argv.slice(2));
+const inputs = argv._;
+const output = argv.output || argv.o;
+const debug = argv.debug;
+const configFile = argv['config-file'];
+delete argv._;
```

Comment:
what's the purpose of deleting these?

Reply:
this deletes the properties out of that are extracted and under control of the script:

```
const inputs = argv._;
const output = argv.output || argv.o;
const debug = argv.debug;
const configFile = argv['config-file'];
```

the remaining arguments are forwarded to the uglify when spawning it via `execFileSync`:

```
  for (arg in argv) {
    const prefix = arg.length == 1 ? '-' : '--';
    const value = argv[arg];
    args.push(prefix + arg);
    if (value && value !== true) {
      args.push(value);
    }
  }

  if (DEBUG) console.error(`Running node ${args.join(' ')}`);

  const isWindows = /^win/i.test(process.platform);
  child_process.execFileSync(
      isWindows ? 'node.cmd' : 'node', args,
      {stdio: [process.stdin, process.stdout, process.stderr]});
```

Will add a comment to make that clearer